.merlin

```diff
@@ -9,4 +9,4 @@ B _build/src/ext/
 B _build/src/ext/pta/
 B _build/src/frontc/
 B _build/src/ocamlutil/
-PKG findlib
+PKG findlib, zarith
```

Comment:
I think we've discussed this before, but I can't find it anymore: why do we still have this manually written `.merlin` file?

I have to always delete it for ocaml-lsp-server to work and pick up the correct ones generated by dune. And then I always have to add it back for `opam pin` because otherwise it fails with some error about the file suddenly being missing.

Reply:
It is needed for the `ocamlbuild` based build. When developing CIL, I usually default to the `./configure && make &&  make test` workflow over the dune-based one, as working with the tests is a lot more convenient in that setting.

Reply:
What kind of additional convenience is there over `dune runtest` then that I am not aware of?

Reply:
The ability to easily run individual tests by changing into the test directory and simply executing `make testname`.

That would probably also work inside the `_build` directory, but then you cannot slightly change a test, rerun with recompiling, make some changes to the source code, compile, rerun again with your still modified test. I find myself doing that a lot when working on CIL.

Reply:
Right, maybe it's just that `./configure` should be creating the `.merlin` file if the old build is used only. But that's a separate problem.